File: src/test/java/me/rasztabiga/fridgy/products/service/mapper/RecipeMapperTest.java
package me.rasztabiga.fridgy.products.service.mapper;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class RecipeMapperTest {

    private RecipeMapper recipeMapper;

    @BeforeEach
    public void setUp() {
        recipeMapper = new RecipeMapperImpl();
    }
}

File: src/test/kotlin/me/rasztabiga/fridgy/products/service/mapper/RecipeMapperTest.kt
package me.rasztabiga.fridgy.products.service.mapper

import org.junit.jupiter.api.BeforeEach

class RecipeMapperTest {

    private lateinit var recipeMapper: RecipeMapper

    @BeforeEach
    fun setUp() {
        recipeMapper = RecipeMapperImpl()
    }
}
